### ISSUE TITLE
[question] slug

### ISSUE THREAD

**Issue by webeks** [scottbedard/blogtags#12]
Hi

Just a quick question - why not adding slug as a unique field in tags table? 
Is there any special reason or just havent had the time?

Thanks,
Miha

**Comment by pvullioud** [comment]
A slug is useful especially for non English languages. We have done a modify version with slugs support.
You can see it in action here
https://navixia.com/en/tags/sensibilisation-a-la-securite-it
Any interest for a PR ?

**Comment by scottbedard** [comment]
@webeks Correct, I've been extremely busy with work and other projects, I haven't been spending much time on this plugin. 

@pvullioud That looks great! And I would certainly accept that PR.

**Comment by wowfrank** [comment]
damn it, as a non-english user, I have to create each tag MANUALLY via database.
could you plz provide some function like create a tag?
